cmd/rook/rook/rook.go

```diff
@@ -63,3 +75,16 @@ func SetupLogging() {
 func NewTableWriter(buffer io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(buffer, outputMinWidth, outputTabWidth, outputPadding, outputPadChar, 0)
 }
+
+func setFlagsFromEnv(flags *pflag.FlagSet) error {
```

Comment:
could this be moved into the util/flags package since it has so much in common with the same code found in rookd? that flags package is an OK dependency for the rook client tool.